octopoes/octopoes/xtdb/client.py

```diff
@@ -46,7 +46,10 @@ class XTDBStatus(BaseModel):
 @functools.cache
 def _get_xtdb_http_session(base_url: str) -> httpx.Client:
     return httpx.Client(
-        base_url=base_url, headers={"Accept": "application/json"}, transport=(httpx.HTTPTransport(retries=3))
+        base_url=base_url,
+        headers={"Accept": "application/json"},
+        transport=(httpx.HTTPTransport(retries=3)),
+        timeout=60,
```

Comment:
I would like to suggest making the httpx timeouts an `env` variable

Reply:
I agree, look for similar options in other places.

Reply:
I was already working on doing that in other places, so I can also include the xtdb client in my PR.